packages/fxa-auth-server/lib/senders/emails/templates/inactiveAccountSecondWarning/en.ftl

```diff
@@ -0,0 +1,9 @@
+inactiveAccountSecondWarning-subject = Action required: Account deletion in 7 days
+inactiveAccountSecondWarning-title = Your { -brand-mozilla } account and data will be deleted in 7 days?
+inactiveAccountSecondWarning-account-description = Your { -product-mozilla-account } is used to access free privacy and browsing products like Firefox Sync, Monitor, Relay, and MDN.</span>
```

Comment:
Remove `</span>` tag (I think that was carried over from my PR - that string (`inactiveAccountFirstWarning-account-description`) will also need to be fixed and updated with new ID 😬) and use brand terms 

```suggestion
inactiveAccountSecondWarning-account-description = Your { -product-mozilla-account } is used to access free privacy and browsing products like { -brand-firefox } sync, { -produt-mozilla-monitor }, { -product-firefox-relay }, and { -product-mdn }.
```

Reply:
Let me know if you'd rather I put up a separate PR to fix the first email, I don't mind!

Reply:
Thanks for including this change!